src/components/Organizations/User/index.tsx

```diff
@@ -0,0 +1,253 @@
+              </p>
+
+              <Footer>
+                <Mutation<{
+                  removeUserFromGroup: {
+                    email: string;
+                    groupId: string;
+                  };
+                }>
+                  mutation={DELETE_USER_FROM_GROUP}
+                >
+                  {(removeUserFromGroup, { error, data }) => {
+                    if (error) {
+                      return <div>{error.message}</div>;
+                    }
+                    if (data) {
+                      refetch();
+                    }
+                    return (
+                      <Button
+                        variant="primary"
+                        action={() => {
+                          void removeUserFromGroup({
+                            variables: {
+                              email: user?.email,
+                              groupId: group?.id,
+                            },
+                          });
+                        }}
+                      >
+                        Continue
+                      </Button>
+                    );
+                  }}
+                </Mutation>
+                <Button variant="ghost" action={closeGroupModal}>
+                  Cancel
+                </Button>
+              </Footer>
+            </Modal>
+          </TableActions>
+        );
+      },
+    },
+  ];
+
+  const UserProjectColumns = [
+    {
+      width: '80%',
+      key: 'name',
+      render: ({ name }: { name: string }) => {
+        return name ? <div className="group">{name}</div> : <>Group name - </>;
+      },
+    },
+
+    {
+      width: '20%',
+      key: 'actions',
+      render: ({ name }: { name: string }) => {
+        return (
+          <TableActions>
+            <ProjectGroupLink
+              className="link"
+              projectGroupSlug={name}
+              organizationSlug={organizationId}
+              organizationName={organizationName}
+              key={name}
+            >
+              <EyeOutlined className="edit" style={{ width: '27px' }} />
+            </ProjectGroupLink>
+          </TableActions>
+        );
+      },
+    },
+  ];
+
+  if (!user) return <p style={{ textAlign: 'center' }}>User not found</p>;
+
+  const userProjects = [...new Set(user.groups.map(g => g.projects).flat())];
+
+  return (
+    <StyledUser>
+      <TableWrapper>
+        <>
+          <PaginatedTable
+            limit={10}
+            data={user.groups}
+            columns={UserColumns}
+            labelText="Groups"
+            emptyText="No groups"
+            disableUrlMutation
+          />
+          <div className="separator" style={{ margin: '3rem 0' }}></div>
+          <PaginatedTable
+            limit={10}
+            data={userProjects}
+            columns={UserProjectColumns}
+            labelText="Projects"
+            emptyText="No projects"
+            disableUrlMutation
+          />
```

Comment:
Displaying projects of a user to an organization owner without any reference of the groups that these projects attached to implies some capability of removing a user from a project directly, and I think it would be best to omit this information.

If you were to display projects, you would want some way to identify the groups that these projects are associated to so that an org owner can quickly see which projects they have access to via the groups they're in, but then you already have this information in the form of the groups that the user has access to, and the groups the projects have access to.

I think there is just too much information to be displaying here and may result in information bloat/overload

Reply:
Mainly because displaying the projects gives you a `quick view` to the project, but you can't actually do anything with the project in this view, you can't remove the user from the project because of technical limitations with how users and projects are associated to groups

and if you want to view a project, you'd go to the projects tab and enter the project there?

Reply:
I think it's important that the owner of the org can view their projects somewhere though, although I agree that linking them via the groups view might make more sense.

I also think that the project needs to link to the project code in the main dashboard, but not imply that the Organization owner is managing it here

Reply:
The current queries for `userByEmailAndOrganization` only returns the users information (name, email, etc) and the `groupRoles` that this user is in. Because access to the projects is assigned via these group roles.

To add a `projects` return to the `userByEmailAndOrganization` query would be very complex, and IMHO it will add too much information to the users page. If the user has 20 groups, and those 20 groups have 20 projects, you have a lot of projects to display. 

Now if you're displaying this many projects for an individual user, what do you gain from this apart from being able to say "yep, this user is in groups that allow them to access 400 projects"? You can't remove them from 1 project directly, you can only view the groups that each project have attached to them. Now if you add "access to project X is via group A, B, C, D, E, F, G, H" to that list, you will eventually hit a point where theres potential for a project to be attached to every single group the user is in, so now you would have a project with a list of 20 groups on it, how do you clearly display this information?

Reply:
>I also think that the project needs to link to the project code in the main dashboard, but not imply that the Organization owner is managing it here

This would only be relevant to the user though, because organization owners/viewers don't have access to the `dashboard/projects/${projectname}` query.